Reduces adding and removing addresses from 2 steps to one step

batangaTroy opened the pull request.

batangaTroy committed:
Reduces 2 steps to one step the process of adding and removing self to/from Registry

pkoch commented on src/components/ActionPage/MiniBackoffice.tsx:
```diff
@@ -30,19 +30,15 @@ export const SingleText = ({ children }: { children: string }) => (
-  await registerUser();
-  await approveUser();
+  await addUserToRegistry();
 };
 
 const removeKYC = async (
-  unRegisterUser: () => Promise<void>,
-  disapproveUser: () => Promise<void>
+  removeUserFromRegistry: () => Promise<void>
 ): Promise<void> => {
-  await disapproveUser();
-  await unRegisterUser();
+  await removeUserFromRegistry();
 };
```
These don't seem to serve a purpose anymore. Let's inline them.

pkoch commented on src/hooks/miniBackoffice.ts:
```diff
@@ -163,15 +161,15 @@ export const useMiniBackoffice = (
     };
   }
 
+  // should we remove this? We should never get here. If so, remove from MiniBackOffice component as well.
```
Yeah, I agree. Let's target only the relevant part of the interface.

pkoch reviewed (commented):
🏓

batangaTroy committed:
Make changes reflected in PR comments

Inlines addKYC and removeKYC and removes extra function call.
Removes unused KYCAbsent state and components.

pkoch reviewed (approved):
Didn't try it out myself, but code looks great! 🚀
